lib/YaST/Firewall/MainPage.pm

```diff
@@ -0,0 +1,68 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: The class introduces all accessing methods for YaST module
+# System Settings Page.
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package YaST::Firewall::MainPage;
+use strict;
+use warnings;
+use testapi;
+
+sub new {
+    my ($class, $args) = @_;
+    my $self = bless {
+        app => $args->{app}
+    }, $class;
+    return $self->init();
+}
+
+sub init {
+    my $self = shift;
+    $self->{overview_tree} = $self->{app}->tree({id => "\"Y2Firewall::Widgets::OverviewTree\""});
+    $self->{btn_accept} = $self->{app}->button({id => 'next'});
+    return $self;
+}
+
+sub is_shown {
+    my ($self) = @_;
+    my $is_shown = $self->{overview_tree}->exist();
+    save_screenshot if $is_shown;
+    return $is_shown;
+
+}
+
+sub select_start_up_page {
```

Comment:
the hyphens can be abbreviated removing them, s/start_up/startup/ to distinguish them from different words. I have seen this in other naming conventions, we  don't have it written anywhere :) but just in case you want to shorten it.

Reply:
In order to keep one style in whole code, i think keep "page" is good.